apps/dashboard/src/main/resources/struts.xml

```diff
@@ -531,6 +531,16 @@
             </result>
         </action>
 
+        <action name="api/redactCollection" class="com.akto.action.ApiCollectionsAction" method="redactCollection">
+            <interceptor-ref name="json"/>
+            <interceptor-ref name="defaultStack" />
+            <result name="SUCCESS"  type="json">
+            </result>
+            <result name="ERROR" type="httpheader">
+                <param name="status">401</param>
```

Comment:
This error code is used for unauthorized requests and redirects the user to login page. Is this the intended use case, if not, we primarilly use 422 for most cases, when we need to send some API specific error message.

Reply:
Fixed